### DISCUSSION TITLE
Finding ways to easily search for OSPOs in a single platform: How to improve the OSPO landscape?

### DISCUSSION THREAD

**Discussion by anajsana** [todogroup/ospology#23] [OSPO Discussions]
This is the OSPO Landscape: https://landscape.todogroup.org/

What features would the OSPO community like to see in the OSPO landscape? Are there any sections/filters that might be missing there?

**Comment by 094459** [comment]
I am a sucker for these visual infographics, so nice job. I was thinking of some categories that might make sense in addition to the ones you have (and might be useful for lurkers/viewers into this community) - the one that come to mind is Consulting (organisations that can help you with all/some aspects of your OSPO).

**Comment by winterrocks** [comment]
If companies have an OSPO contact (ospo@company.com or a person possibly) it might be useful as today in many cases it is difficult/impossible to identify who is the OSPO contact person. Of course, nobody wants to be exposed to an extensive number of emails or spam, so it might be a bit difficult.

**Comment by ashleywolf** [comment]
We discussed this briefly in the TODO Group Work Group. There was an effort to capture more OSPO tooling at https://github.com/todogroup/awesome-ospo. Perhaps we decide how and when to use each would be useful.

**Reply by 094459** [reply]
> We discussed this briefly in the TODO Group Work Group. There was an effort to capture more OSPO tooling at https://github.com/todogroup/awesome-ospo. Perhaps we decide how and when to use each would be useful.

I would like to see this, the tooling split out into the different focus areas.

**Comment by winterrocks** [comment]
Currently, the Industries filter is flat and has many industries, maybe there could be some hierarchy added there? I know, some companies may fall in several industries, but for those, it should be possible to have many industries tags, so that they show up in several places.
